docs/tests.rst

```diff
@@ -59,7 +59,7 @@ Via Docker with a graphical instance of Firefox (Linux)
   $ docker-compose -f docker-compose.test.yml run --rm acceptance
 
 
-Via Docker with a graphical instance of Firefox (macOS)
+Via Docker with a graphical instance of Firefox (MacOS)
```

Comment:
(`macOS` is the correct capitalization: https://www.apple.com/lae/macos/sierra/)